packages/teleterm/src/ui/TabHost/ClusterConnectPanel/ClusterConnectPanel.tsx

```diff
@@ -14,18 +14,17 @@ export function ClusterConnectPanel() {
   return (
     <Flex
       m="auto"
-      width="400px"
+      width="370px"
       pb={6}
       flexDirection="column"
       alignItems="center"
     >
       <Image width="120px" src={clusterPng} mb={3} />
       <Text typography="h3" bold mb={2}>
-        Connect to a Cluster
+        Connect a Cluster.
```

Comment:
The dots look a bit weird for me, but I asked Xin about them already.

Reply:
I think, you're right, removed.